.github/workflows/test_size.yml

```diff
@@ -43,16 +43,6 @@ jobs:
           path: ~/.ccache
           key: ${{github.workflow}}-ccache-${{ matrix.toolchain }}-${{steps.ccache_cache_timestamp.outputs.timestamp}}
           restore-keys: ${{github.workflow}}-ccache-${{ matrix.toolchain }}-  # restore ccache from either previous build on this branch or on master
-      - name: setup ccache
```

Comment:
@tridge why removing the ccache config here ? error ?

Reply:
the common setup doesn't work as its not yet in master, and this is operating on master branch. Once this PR is in master it could then be put back in